server.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var port = process.env.PORT || 3000;
+
+var express = require("express");
+var Words = require("./lib/words.js");
+var bdprs = require('body-parser');
+
+var app = express();
+var w = new Words();
+var adj = new w.adj();
```

Comment:
wherever possible, use actual words instead of abbreviations as your variable names. It makes others (and your future self) happy. For example - I have no idea what nn is or does.